common/app/templates/inlineJS/blocking/applyRenderConditions.scala.js

```diff
@@ -101,5 +140,10 @@
     if (baseFontSize && parseInt(baseFontSize, 10) !== 16) {
         documentElement.style.fontSize = baseFontSize
     }
+
+    if (hideReaderRevenue()) {
+        docClass += ' hide-reader-revenue';
```

Comment:
.... or that. You might need the mixin though if it was anything more complicated than simply hiding